lib/core/src/sdk.rs

```diff
@@ -556,6 +557,7 @@ impl LiquidSdk {
             let mut current_bitcoin_block: u32 = 0;
             let mut shutdown_receiver = cloned.shutdown_receiver.clone();
             let mut interval = tokio::time::interval(Duration::from_secs(10));
+            #[cfg(not(all(target_family = "wasm", target_os = "unknown")))]
             interval.set_missed_tick_behavior(MissedTickBehavior::Skip);
```

Comment:
Is it not possible to use the `interval` from the vanilla tokio so that we can set the missed tick behavior?

Reply:
The `Interval` in this case is specific to WASM, using javascripts setTimeout/clearTimeout internally